azavea/noaa-hydro-data
# EXPERIMENTAL: Add, configure nbdev

## Overview

Allows for git-friendly Jupyter development.

See https://nbdev.fast.ai/01_Tutorials/02_git_friendly_jupyter.html and https://www.fast.ai/2022/08/25/jupyter-git/

### Checklist

- [ ] Ran `nbautoexport export .` in `/opt/src/notebooks` and committed the generated scripts. This is to make reviewing notebooks easier. (Note the export will happen automatically after saving notebooks from the Jupyter web app.)

## Testing Instructions

- Run ./scripts/setup
- Change a notebook and commit the changes locally
  - [ ] Ensure only meaningful changes are saved, not metadata or execution order
